### PR TITLE
modifiy edgehub metamanager log to help reading

### PR DESCRIPTION
**Pull request by GsssC**
**What type of PR is this?**
log modification
**What this PR does / why we need it**:
because there are multiple `process.go` such as MetaManger/process.go, EdgeHub/process.go, etc.
It is necessary to distinguish them more intuitivly in log.

### CODE REVIEW

File: edge/pkg/edgehub/process.go [hunk 1]
```diff
@@ -154,15 +154,15 @@ func (eh *EdgeHub) routeToEdge() {
...
 		message, err := eh.chClient.Receive()
 		if err != nil {
-			klog.Errorf("websocket read error: %v", err)
+			klog.Errorf("EdgeHub websocket read error: %v", err)
 			eh.reconnectChan <- struct{}{}
 			return
 		}
 
-		klog.Infof("received msg from cloud-hub:%+v", message)
+		klog.V(2).Infof("EdgeHub received msg from cloud-hub: %+v", message)
```
Comment (kadisi) [inline edge/pkg/edgehub/process.go:162]: why use V2?

Comment (GsssC) [reply edge/pkg/edgehub/process.go:162]: Message here will be logged by MetaManager or other modules as well. For my view, edgehub would better be transparent and only to log error.

**Comment by daixiang0** [comment]
Is there any possible to add prefix to each component log like `[edgehub] xxxx`?

**Comment by fisherxu** [comment]
> Is there any possible to add prefix to each component log like [edgehub] xxxx?
It would be nice and helpful to locate issue.

Good idea :)

File: edge/pkg/edgehub/process.go [hunk 2]
```diff
@@ -172,8 +172,8 @@ func (eh *EdgeHub) sendToCloud(message model.Message) error {
 	err := eh.chClient.Send(message)
 	eh.keeperLock.Unlock()
 	if err != nil {
-		klog.Errorf("failed to send message: %v", err)
-		return fmt.Errorf("failed to send message, error: %v", err)
+		klog.Errorf("EdgeHub client failed to send message to cloud: %v", err)
+		return fmt.Errorf("edgeHub client failed to send message to cloud, error: %v", err)
```
Comment (fisherxu) [inline edge/pkg/edgehub/process.go:176]: EdgeHub?

**Comment by GsssC** [comment]
@daixiang0 I can’t agree more. I just add prefix to some important  and periodic log manually.

**Comment by fisherxu** [comment]
> @daixiang0 I can’t agree more. I just add prefix to some important and periodic log manually.

So we can add the prefix for some logs and add log level for some meaningless logs.

**Pull request closed without merging**